libgadget/run.c

```diff
@@ -39,7 +39,7 @@ void init(int snapnum, DomainDecomp * ddecomp); /* init.c only used here */
  * reached, when a `stop' file is found in the output directory, or
  * when the simulation ends because we arrived at TimeMax.
  */
-static void compute_accelerations(int is_PM, int FirstStep, int GasEnabled, int HybridNuGrav, ForceTree * tree, DomainDecomp * ddecomp);
+static void compute_accelerations(const ActiveParticles * act, int is_PM, PetaPM * pm, int FirstStep, int GasEnabled, int HybridNuGrav, ForceTree * tree, DomainDecomp * ddecomp);
```

Comment:
If the step is not PM, can we pass in pm == NULL instead?

Reply:
I don't think so, we still need it to tell the short-range gravity what to do.